demos/polls/sql/install.sh

```diff
@@ -5,11 +5,14 @@ case "${unameOut}" in
     *)          pg_cmd="sudo -u postgres psql"
 esac
 
-${pg_cmd} -c "DROP DATABASE IF EXISTS aiohttpdemo_polls"
-${pg_cmd} -c "DROP ROLE IF EXISTS aiohttpdemo_user"
-${pg_cmd} -c "CREATE USER aiohttpdemo_user WITH PASSWORD 'aiohttpdemo_user';"
-${pg_cmd} -c "CREATE DATABASE aiohttpdemo_polls ENCODING 'UTF8';"
-${pg_cmd} -c "GRANT ALL PRIVILEGES ON DATABASE aiohttpdemo_polls TO aiohttpdemo_user;"
+pg_cmd+=" -h $DB_HOST -p $DB_PORT"
```

Comment:
AFAIR bash does not have such operator. Did you test this? I bet it'd fail.

Reply:
Tested both on mac and ubuntu.
http://tldp.org/LDP/abs/html/bashver3.html#PATHAPPEND